app/views/merchant/items/index.html.erb

```diff
@@ -5,13 +5,42 @@
 <% @items.each do |item| %>
   <div id="item-<%= item.id %>">
     <h3><%= item.name %></h3>
-    Description: <%= item.description %></br>
-    Price: <%= item.unit_price %></br>
-    Status: <%= item.status %></br>
+    <p>Description: <%= item.description %></p>
```

Comment:
We'll need to refactor this so that it's just two sections: enabled/disabled and each item has all of the information attached to it within that section.  I'll refactor this.